src/libraries/System.Runtime/tests/System.Runtime.Tests/System.Runtime.Tests.csproj

```diff
@@ -17,6 +17,9 @@
     <!-- Active issue: https://github.com/dotnet/runtime/issues/97809 -->
     <ShouldILStrip>false</ShouldILStrip>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetOS)' == 'browser'">
+    <XunitShowProgress>true</XunitShowProgress>
+  </PropertyGroup>
```

Comment:
this should be reverted

Reply:
I would like to keep it in, just in case the issue reappear, to have more information about which test is causing it.

Reply:
We recently removed the progress printing since it was thought to cause timeouts: https://github.com/dotnet/runtime/pull/103621